themes-default/slim/src/components/app-header.vue

```diff
@@ -9,7 +9,11 @@
                     <span class="icon-bar" />
                     <span class="icon-bar" />
                 </button>
-                <app-link class="navbar-brand" href="home/" title="Medusa"><img alt="Medusa" src="images/medusa.png" style="height: 50px;" class="img-responsive pull-left"></app-link>
+                <app-link v-if="isConnected || !socketIsConnected" class="navbar-brand" style="position: relative" title="Medusa">
```

Comment:
Isn't this supposed to be `!isConnected`?

Right now I always get a disconnected icon and so clicking on the logo refreshes the page.

Reply:
I fixed that in a PR that came after

Reply:
Was broken on my end. Had the fixed version, but my Synology Reverse Proxy didn't forward it correctly without some additional config.

https://mlohr.com/websockets-for-synology-dsm/